src/main/resources/application.conf

```diff
@@ -5,6 +5,7 @@ authorization-server {
   ec-keys-identifiers = ${?EC_KEYS_IDENTIFIERS}
   rsa-keys-identifiers = ${?RSA_KEYS_IDENTIFIERS}
   jwt-queue-url = ${JWT_QUEUE_URL}
+  jwt-fallback-bucket = ${JWT_FALLBACK_BUCKET}
```

Comment:
this should be added also in `application-standalone.conf` (maybe with a dummy value to avoid to accidentally write on a real bucket)